### PR TITLE
connected database, completed userpost model, and completed seeds for…

### PR DESCRIPTION
**Pull request by lavanyavirushan**
connected database, completed userpost model, and completed seeds for users, categories and user post.
lets not track .env file in github rather have empty values in .env.sample file

### CODE REVIEW

**Review by MadelineLowes — APPROVED** [review]
looks good

File: models/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const User = require('./User');
 const Category = require('./Category');
-const UserPost = require('./UserPost');
+const UserPost = require('./Post');
```
Comment (PhilipVKurian) [inline models/index.js]: userpost

**Review by PhilipVKurian — COMMENTED** [review]
looks good

**Review by MadelineLowes — APPROVED** [review]
looks good

**Pull request merged**